icu4j/main/localespi/pom.xml

```diff
@@ -135,22 +135,22 @@
         <configuration>
           <includes>
             <!-- treat localespi tests as integration tests because they depend on .jar files from package phase -->
-            <include>**/*.java</include>
+            <include>**/*Test.java</include>
```

Comment:
Do we need to make this more specific? We have run into trouble with that before...

Reply:
In this case it does not really matter, all files in `icu4j\main\localespi\src\test\java\com\ibm\icu\dev\test\localespi` follow the `*Test.java` convention. Except for `TestUtil.java`, an utility class that does not have `@RunWith`

---

In the longer run this deserves a cleanup.

Maven is very much convention based.

`*Test` and `Test*` are test classes, `IT*`, `*IT` and `*ITCase` are integration test classes.
https://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html

Everything is in `localespi` is integration test, but uses bad names (`*Test`), that's why explicitly force them in)

If we rename these files to `*IT` we can remove all the `<includes>` in the plugin.

---

Optional: I can rename them now, and remove the `includes`.

Reply:
At least for now I prefer to keep the class names and use only one include that captures them all.

I like `*.java` because it's foolproof. It seems super easy to not know some naming convention and accidentally create a test class that is silently ignored.

Reply:
Done, back to *.java